src/friendly-iframe-embed.js

```diff
@@ -414,7 +421,7 @@ export class FriendlyIframeEmbed {
    * Notice that this signal coincides with the embed's `render-start`.
    * @return {!Promise}
    */
-  whenReady() {
+  whenRenderStarted() {
```

Comment:
I see a couple of places where the old `whenReady` is still used. E.g. in [amp-animations](https://github.com/ampproject/amphtml/blob/35d3569808d968261c6a0209951ccf0828c6728f/extensions/amp-animation/0.1/amp-animation.js#L485). Notably, amp-animations should be using `whenRenderComplete()` now. Not sure if our compiler will indicate to you all other possible usages.

Reply:
good catch, switched.